From bb5aca68e17eb38006fb2ce5985d32bbf39a7ac9 Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Thu, 3 Aug 2017 11:19:21 +0800
Subject: [PATCH 2280/5242] MLK-16131 arm64: dts: freescale: imx8mq: disable
 pwm led for over-drive mode

commit  f05372d4b43ab16e68f93997497249fb6b25b413 from
https://source.codeaurora.org/external/imx/linux-imx.git

On i.MX8MQ EVK A1 board, GPIO1_IO13 pin is shared between
GPIO regulator and PWM2, when using over-drive dtb, there
will be no PWM2 function, disable PWM LED to avoid below
message during suspend/resume test:

[   19.754862] PM: resume of devices complete after 118.732 msecs
[   19.761740] Restarting tasks ...
[   19.761741] leds_pwm pwmleds: unable to request PWM for PWM2: -517
[   19.773557] done.

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../arm64/boot/dts/freescale/fsl-imx8mq-evk-od.dts |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-od.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-od.dts
index 0f4574d..ac1b658 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-od.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-od.dts
@@ -29,6 +29,10 @@
 			states = <900000 0x1 1000000 0x0>;
 		};
 	};
+
+	pwmleds {
+		status = "disabled";
+	};
 };
 
 &pwm2 {
-- 
1.7.9.5

